SPECS/systemd/systemd.spec

```diff
@@ -174,6 +178,12 @@ install -D -m 0644 %{SOURCE4} %{buildroot}%{_libdir}/systemd/system-preset/99-ma
 %find_lang %{name} ../%{name}.lang
 
 %check
+# Generate machine-id using dbus
+%{_bindir}/dbus-uuidgen --ensure=%{_sysconfdir}/machine-id
```

Comment:
Tagging @ddstreet to take a look.

Reply:
There were 11 test-cases which were failing because there was no machine-id. Will look for a workaround because it significantly increases the pass rate